topics/ecology/tutorials/Ecoregionalization_tutorial/tutorial.md

```diff
@@ -0,0 +1,352 @@
+---
+layout: tutorial_hands_on
+
+title: Ecoregionalization workflow tutorial
+questions:
+- How to use the ecoregionalization workflow?
+- How to construct ecoregions from occurrences species data and environemental data? 
+objectives:
+- Learning how to use the ecoragionalization workflow.
+- Learning about boosted regression trees algorithm.
+- Learning about Sihlouette index and select an appropriate number of cluster
+- Learning about Clara clustering algorithm (PAM for large dataset)
+- Create an ecoregionalization map
+time_estimation: 1H
+key_points:
+- BRT tool
+- TaxaSeeker tool
+- ClusterEstimate tool
+- ClaraClust tool
+- EcoMap tool
+tags:
+- ecology
+- EBV workflow
+- modeling
+contributors:
+- PaulineSGN
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.